demo/NumberControl/NumberDemo/src/buildInteractionModel.ts

```diff
@@ -33,6 +34,8 @@ export namespace BasicNumberDemoIM {
 
 // If launched directly, build and write to a file
 if (require.main === module) {
+    // get logger
```

Comment:
nit: these ones (in IMgen scripts) don't have to move.  they could stay at the top so long as they come after setting the custom services.  But I have no problem with them getting moved either.

Reply:
Reverted back to stay on top.